packages/react-jss/src/createHoc.js

```diff
@@ -223,22 +214,22 @@ export default function createHOC<
       return {
         staticSheet,
         dynamicSheet,
-        classes: this.computeClasses(staticSheet, dynamicSheet)
+        classes: mergeClasses(staticSheet.classes, dynamicSheet ? dynamicSheet.classes : {})
```

Comment:
could this benefit from memoized mergeClassesProp? its also called on each update

Reply:
No this is only called when creating a new stylesheet, so only on theme changes and on mount when no sheet exists